config/environments/production.rb

```diff
@@ -29,7 +29,7 @@
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Use log level "warn" in prod to avoid logging parameters.
-  config.log_level = :warn
+  config.log_level = ENV['CONJUR_LOG_LEVEL'] || :warn
```

Comment:
Is the ENV used for OSS production?  If so, why would the level be different from the appliance?

Reply:
good call, missed that file. added to new PR